Editor/EditorCore/UVEditor.cs

```diff
@@ -2272,6 +2272,23 @@ Bounds2D UVSelectionBounds()
             return new Bounds2D(new Vector2((xMin + xMax) / 2f, (yMin + yMax) / 2f), new Vector2(xMax - xMin, yMax - yMin));
         }
 
+        /// <summary>
+        /// Returns the miniaml u and v values of the current selection in UV space
+        /// </summary>
+        /// <returns></returns>
+        internal Vector2 UVSelectionLowerLeftUV()
+        {
+            Vector2 minimalUV = new Vector2();
+            minimalUV = Vector2.zero;
```

Comment:
Why not initialize directly Vector2 with Vector2.zero from declaration?

Reply:
Indeed changed